### PR TITLE
feat: add .gitattributes

### PR DESCRIPTION
**Pull request by Zeroks77**
I have just created a .gitattributes for handling the line ends.

Did I miss some settings? 

ISSUES CLOSED: #261

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* text=auto
+.gitattributes text eol=lf
+.gitignore text eol=lf
```
Comment (DanielHabenicht) [inline .gitattributes]: should be sufficient without these two lines. Or am I missing something?

Comment (mschwrdtnr) [reply .gitattributes]: Why do you think so? `* text=auto` -> *Git will handle the files in whatever way it thinks is best. This is a good default option.*
So the best option isn't always `eol=lf` and so we need line 2.
I don't really get what the difference between line 2 and line 3 is and why you need `.gitattributes` and ´.gitignore´ @Zeroks77 ?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+# uses the optimal lineending but prioritize lf
+* text=auto elo=lf
```
Comment (mschwrdtnr) [inline .gitattributes]:
```suggestion
* text=auto eol=lf
```
??

Comment (DanielHabenicht) [reply .gitattributes]: good catch

Comment (Zeroks77) [reply .gitattributes]: my bad sry

**Comment by DanielHabenicht** [comment]
@Zeroks77 by the way. This is not a feature it's is `chore` as it does not change the application for the user.

**Comment by mschwrdtnr** [comment]
> @Zeroks77 by the way. This is not a feature it's is `chore` as it does not change the application for the user.

My mistake

**Pull request merged**